kernel/include/timeout_q.h

```diff
@@ -48,6 +48,8 @@ static inline int _abort_thread_timeout(struct k_thread *thread)
 
 s32_t _get_next_timeout_expiry(void);
 
+void _set_timeout_expiry(s32_t ticks, bool idle);
```

Comment:
Nitpick: Should be "z_set_timeout_expiry()" if it's not a static.  We're not allowed to use prepended underscores anymore.  Probably also wants documentation pointing out that this is the kernel API that wraps z_clock_set_timeout(), and that the driver API shouldn't be used outside the framework.

Reply:
Fixed name